rules/int_crawler.go

```diff
@@ -133,7 +134,8 @@ func (ic *intCrawler) run() {
 			}
 		}
 		logger.Info("Crawler run complete")
-		for i := 0; i < ic.interval; i++ {
+		intervalSeconds := int(ic.interval.Generate().Seconds())
```

Comment:
Maybe debug log the seconds here to see the jitter for crawler?

Reply:
Added in 0776601